api/data_explorer/__main__.py

```diff
@@ -122,10 +122,12 @@ def _get_table_names():
 
 
 def _get_facets():
-    """Gets Elasticsearch facets.
+    """Gets facets.
 
     Returns:
-        A dict from UI facet name to Elasticsearch facet object
+        A list of tuples, where each tuple corresponds to a facet. Each tuple
```

Comment:
I could be missing something here, but are there any use cases that require us to have both the UI facet description and the es facet object? If not, we could leave `app.config['ELASTICSEARCH_FACETS']` as a dict of facet names to es objects and separately have an `app.config['UI_FACETS']` that's either a list or a dict of `Facet` objects.

That would let us keep the ES dict where it's useful in the controller and search instead of having to re-construct it and search through lists while also making these changes extensible if we ever want to expand the API definition of a UI Facet.

Reply:
Done